### DISCUSSION TITLE
Adding customhostgroup/customportgroup #10 and more in Web interface

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#12429] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hey!

Small question: we ran out of available custom host/port groups in web interface (due to many custom syslog streams). 
Added new groups in files:

- local/salt/firewall/defaults.yaml;
- local/salt/firewall/soc_firewall.yaml.

Restarted firewall, nginx, soc, but can't see them in web interface.
How can I add them to web interface to manage them?

Thanks!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Sahale** [comment]
Still didn't make it.
Any advice?

**Reply by cm-ops** [reply]
This is being looked at to add more custom configurations to SOC, firewall groups included.

**Reply by Sahale** [reply]
Ok, will be waiting.
